selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -199,6 +200,15 @@ applyCustomOptions() {
     done
 }
 
+extractSystemProperties() {
+    for var in "$@"; do
+        if [[ "$var" =~ -D.* ]]; then
+            MAVEN_OPTIONS=${MAVEN_OPTIONS}" "$var
```

Comment:
From the readability point it's better to enclose variables into the ", that is to have:
```"${MAVEN_OPTIONS} $var"```